### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

File: pages/configuration/https.rst [hunk 1]
```diff
@@ -44,7 +44,7 @@ Create PKCS#5 and X.509 certificate::
   Country Name (2 letter code) [AU]:DE
   State or Province Name (full name) [Some-State]:Hamburg
   Locality Name (eg, city) []:Hamburg
-  Organization Name (eg, company) [Internet Widgits Pty Ltd]:Graylog, Inc.
+  Organization Name (eg, company) [Internet Widgets Pty Ltd]:Graylog, Inc.
```
Comment (joschi) [inline pages/configuration/https.rst]: This is actually "correct", as it's the default in OpenSSL:
https://github.com/openssl/openssl/blob/OpenSSL_1_1_0d/apps/openssl.cnf#L139

File: pages/queries.rst [hunk 2]
```diff
@@ -74,12 +74,12 @@ See `wildcard and regexp queries <https://www.elastic.co/guide/en/elasticsearch/
 **Fuzziness:** You can search for similar but not equal terms::
 
   ssh logni~
-  source:exmaple.org~
+  source:example.org~
```
Comment (joschi) [inline pages/queries.rst]: This was intentionally spelled wrong to demonstrate fuzzy searches.

File: pages/queries.rst [hunk 3]
```diff
@@ -74,12 +74,12 @@ See `wildcard and regexp queries <https://www.elastic.co/guide/en/elasticsearch/
...
 
 This is using the `Damerau–Levenshtein distance <http://en.wikipedia.org/wiki/Damerau-Levenshtein_distance>`_ with a default
 distance of *2*. You can change the distance like this::
 
-  source:exmaple.org~1
+  source:example.org~1
```
Comment (joschi) [inline pages/queries.rst]: This was intentionally spelled wrong to demonstrate fuzzy searches.

**Review by joschi — CHANGES_REQUESTED** [review]
@jsoref Thanks for your contribution to Graylog!

There are a few things to address in this PR before we can merge it.

File: pages/configuration/https.rst (hunk 1 above)
Comment (jsoref) [reply pages/configuration/https.rst]: Ah. Right. I've been meaning to clean that project up... Maybe next week...

File: pages/queries.rst (hunk 2 above)
Comment (jsoref) [reply pages/queries.rst]: Would you be ok w/ including ("example is intentionally misspelled below") or something?
(That wouldn't be a change I'd make in this PR, I will drop these two commits when I'm next at my personal workstation.)

Comment (joschi) [reply pages/queries.rst]: Sure, adding an explanation to these search queries would be fine.

**Review by joschi — APPROVED** [review]

**Comment by joschi** [comment]
@jsoref Thanks!

**Pull request merged**